js/MVP/views/landing/components/MotebehovPanel.js

```diff
@@ -0,0 +1,150 @@
+import AlertStripe from 'nav-frontend-alertstriper';
+import { Knapp } from 'nav-frontend-knapper';
+import Lenke from 'nav-frontend-lenker';
+import ModalWrapper from 'nav-frontend-modal';
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import ButtonLenke from '../../../components/ButtonLenke';
+import DialogmotePanel from '../../../containers/DialogmotePanel';
+import { skjemaTypes } from '../../../globals/constants';
+import { MOTEBEHOV_URL, OPPFOLGINGSPLANER_URL, statiskeURLer } from '../../../globals/paths';
+import MotebehovKvittering from './Motebehov/MotebehovKvittering';
+
+const DialogmotePanelStyled = styled(DialogmotePanel)`
+  margin-bottom: 32px;
+`;
+
+const TekstomradeStyled = styled.p`
+  margin: 32px 0;
+`;
+
+const AlertstripeStyled = styled(AlertStripe)`
+  margin: 32px 0;
+`;
+
+const texts = {
+  title: 'Trenger dere et dialogmøte med NAV?',
+  titleSvarBehov: 'Trenger dere et dialogmøte med NAV?',
+  button: 'Vurder behov for møte',
+  text1: `Målet med et dialogmøtet er å oppsummere hva som har skjedd til nå, og snakke om hva som kan hjelpe deg å komme tilbake til arbeid.`,
+  text2: `Ønsker du å snakke med NAV om sykepenger eller noe annet, kan du `,
+  link: 'gå hit for å kontakte oss på andre måter.',
+  titleSvartSvarBehov: 'Du har svart på om du ønsker et møte',
+  titleSvart: 'Du har svart på om du ønsker et møte',
+  buttonSvart: 'Se svaret ditt',
+  textSvart: 'Vi vil bruke svaret ditt når vi vurderer om det er nødvendig med dialogmøte.',
+  alertstripe: 'Husk å dele oppfølgingsplanen med NAV før møtet.',
+  oppfolgingsplanlink: 'Gå til oppfølgingsplanen.',
+};
+
+const text = () => {
```

Comment:
Kanskje rename "text" til f.eks "ContactInfo" eller noe.

Kan vurdere å bruke stor forbokstav i funksjonen og kalle den som en functional component. (ala <ContactInfo />).
Kanskje eksportere til egen fil også, bare for å skille ut litt.

Reply:
Yepp! Enig i det ✅ 

Tror vi kan trekke ut flere komponenter. Etter deploy?